README.md

```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
 ninja install-c2s
 ```
 
-### Deployment
+Note: For computers with 16G and below memory, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.
```

Comment:
For computers with 16G or less memory, we recommend using 4-6 threads for compilation. Compiling SYCLomatic with too many threads may cause out of memory.